packages/medusa/src/api/routes/store/variants/index.ts

```diff
@@ -27,7 +27,7 @@ export default (app) => {
   return app
 }
 
-export const defaultStoreVariantRelations = ["prices", "options"]
+export const defaultStoreVariantRelations = ["prices", "options", "product"]
```

Comment:
@riqwan - check if product is needed here.

Reply:
comment: I imagine storefront builder will use properties from the parent product, when retrieving variants directly, so would keep this in.